ZAPD/Main.cpp

```diff
@@ -35,18 +35,25 @@ void BuildAssetBlob(const fs::path& blobFilePath, const fs::path& outPath);
 #define ARRAY_COUNT(arr) (sizeof(arr) / sizeof(arr[0]))
 void ErrorHandler(int sig)
 {
-	void* array[4096];
-	const size_t nMaxFrames = sizeof(array) / sizeof(array[0]);
+	void* array[0x1000];
+	const size_t nMaxFrames = ARRAY_COUNT(array);
 	size_t size = backtrace(array, nMaxFrames);
 	char** symbols = backtrace_symbols(array, nMaxFrames);
 
 	fprintf(stderr, "\nZAPD crashed. (Signal: %i)\n", sig);
 
 	// Feel free to add more crash messages.
 	const char* crashEasterEgg[] = {
+		"\tIT'S A SECRET TO EVERYBODY. \n\tBut it shouldn't be, you'd better ask about it!",
+		"\tI AM ERROR.",
+		"\tGRUMBLE,GRUMBLE...",
+		"\tDODONGO DISLIKES SMOKE \n\tAnd ZAPD dislikes whatever you fed it.",
+		"\tMay the way of the Hero lead \n\tto the debugger."
+		"\tTHE WIND FISH SLUMBERS LONG... \n\tTHE HERO'S LIFE GONE... "
+		"\tSEA BEARS FOAM, SLEEP BEARS DREAMS. \n\tBOTH END IN THE SAME WAY CRASSSH!"
 		"\tYou've met with a terrible fate, haven't you?",
-		"\tSEA BEARS FOAM. SLEEP BEARS DREAMS. \n\tBOTH END IN THE SAME WAY: CRASSSH!",
-	};
+		"\tMaster, I calculate a 100% probability that ZAPD has crashed. \n\tAdditionally, the "
+		"batteries in your Wii Remote are nearly depleted."};
```

Comment:
This is a very important change, to which I would suggest adding the now famous:
```
    CONGRATURATIONS!    
All Pages are displayed.
       THANK YOU!       
 You are great debugger!"
 ```